src/components/AppWorkspace.vue

```diff
@@ -15,6 +15,31 @@
             activeTabComponent() {
                 return this.activeTab.toLowerCase() + "-tab"
             },
+        },
+
+        data() {
+            return {
+                saveDebounce: null
+            }
+        },
+
+        mounted() {
+            const debounce = (func, delay) => {
+                return function() {
+                    const context = this
+                    const args = arguments
+                    clearTimeout(this.saveDebounce)
+                    this.saveDebounce = setTimeout(() => func.apply(context, args), delay)
+                }.bind(this)
+            }
+
+            this.$store.subscribe((mutation, state) => {
+                // Assume each mutation produces a new state - no need for hash
+                debounce(async () => {
+                    let result = await this.$store.dispatch('save')
+                    console.log(result)
+                }, 3500)()                
```

Comment:
Move this magic number to options.
And I guess it saves every 3.5 sec instead of 5? 🙈

Reply:
Good idea! On the composer package we can afford high update frequency, on pipedream.ai lower

Reply:
Fixed